app/authorization/client/views/permissions.js

```diff
@@ -94,18 +108,16 @@ Template.permissions.helpers({
 		const filter = new RegExp(s.escapeRegExp(state.get('filter')), 'i');
 		return ChatPermissions.find(
 			{
-				_id: filter,
 				level: CONSTANTS.SETTINGS_LEVEL,
 				group: { $exists: true },
 			},
 			{
-				limit,
 				sort: {
 					group: 1,
 					section: 1,
 				},
 			},
-		);
+		).fetch().map((permission) => ({ ...permission, name: permissionName(permission) })).filter((permission) => filter.test(permission.name) || filter.test(permission._id)).slice(0, limit);
```

Comment:
You removed `limit` from one of the queries, and kept `const limit = state.get('size');`. Could you take a look please? thanks!